plugins/tedge_docker_plugin/tedge_docker_plugin.sh

```diff
@@ -15,8 +15,8 @@ EOF
 }
 
 unsupported_args_check() {
-    if ! [ -z $1 ]; then
-        echo "Unsupported arguments: $@"
+    if [ -n "$1" ]; then
+        echo "Unsupported arguments: $*"
```

Comment:
What is this change?

Reply:
This is [SC2145](https://www.shellcheck.net/wiki/SC2145). In short: `$@` is array access in bash, `$*` is "all arguments as a string".